projects/igniteui-angular/src/lib/core/styles/components/card/_card-theme.scss

```
@@ -188,7 +167,10 @@
         background: --var($theme, 'background');
         transition: $card-transitions;
         backface-visibility: hidden;
-        border: 1px solid --var($theme, 'card-border-color');
+
+        @if map-get($theme, variant) == 'bootstrap' or map-get($theme, variant) == 'fluent' {
```

I see this check in a lot of places throughout many of the components. It would be better if we store the theme variant in a variable at the beginning fo the theme so we don't have to constantly call map-get. It will also make the checks more readable, for instance:
```suggestion
        @if $variant is 'bootstrap' or $variant is 'fluent' {
```